Козаева Яна Алексадровна P3120 task-3

yanakozaeva opened the pull request:
modified:   src/main/cpp/t01_quad.cpp
	modified:   src/main/cpp/t02_divisor.cpp
	modified:   src/main/cpp/t03_twos.cpp
	modified:   src/main/cpp/t04_sum.cpp
	modified:   src/main/cpp/t05_max.cpp
	modified:   src/main/cpp/t06_max_count.cpp
	modified:   src/main/cpp/t07_max_2.cpp
	modified:   src/main/cpp/t08_fibb.cpp
	modified:   src/main/cpp/t09_row.cpp
	modified:   src/main/cpp/t10_max_local.cpp

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t01_quad.cpp
	modified:   src/main/cpp/t02_divisor.cpp
	modified:   src/main/cpp/t03_twos.cpp
	modified:   src/main/cpp/t04_sum.cpp
	modified:   src/main/cpp/t05_max.cpp
	modified:   src/main/cpp/t06_max_count.cpp
	modified:   src/main/cpp/t07_max_2.cpp
	modified:   src/main/cpp/t08_fibb.cpp
	modified:   src/main/cpp/t09_row.cpp
	modified:   src/main/cpp/t10_max_local.cpp

thejerome commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912&bid=15807325)

thejerome commented on src/main/cpp/t09_row.cpp:
```diff
@@ -23,5 +23,33 @@
+		if (a != numb)
+		{
+		count = 1;
+			count=1;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'count' is reassigned a value before the old one has been used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)

thejerome commented on src/main/cpp/t03_twos.cpp:
```diff
@@ -23,5 +23,28 @@
 int t03_twos() {
+	int n;
+	cin >> n;
+	int s = 2;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 's' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)

thejerome commented on src/main/cpp/t10_max_local.cpp:
```diff
@@ -24,5 +24,24 @@
+	int b = 1;
+	int count = 0;
+	cin >> a;
+	int o = 0;
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'o' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734912)

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t03_twos.cpp
	modified:   src/main/cpp/t09_row.cpp
	modified:   src/main/cpp/t10_max_local.cpp

yanakozaeva committed:
Changes to be committed:
	modified:   src/main/cpp/t06_max_count.cpp
	modified:   src/main/cpp/t07_max_2.cpp

thejerome reviewed (commented):
Score: **80**
